server/app/models/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
I don't think this Line is necessary
Node is in strict mode

Reply:
This file was automatically generated by sequelize-cli. Would you suggest I remove it now or maybe later in a branch for refactoring.

Reply:
Ok boss